### PR TITLE
Add contributing guidelines

### LINKED ISSUE #832: Update README
* specs are frozen and immutable
* clarify contributing guidelines to `cw-plus` and the possibility of proposing new specs

### PR DESCRIPTION
**Pull request by uint**
Closes #832

Add contributing guidelines, freeze specs, format .md

The rich diff is much easier to view.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,102 +25,80 @@
...
-your projects.
+**Warning** None of these contracts have been audited and no liability is assumed for the use of this code. They are
+provided to turbo-start your projects.
 
-**Note** All code in pre-1.0 packages is in "draft" form, meaning it may
-undergo minor changes and additions until 1.0. For example between 0.1 and
-0.2 we adjusted the `Expiration` type to make the JSON representation
-cleaner (before: `expires: {at_height: {height: 12345}}` after
-`expires: {at_height: 12345}`)
+**Note** All code in pre-1.0 packages is in "draft" form, meaning it may undergo minor changes and additions until 1.0.
```
Comment (ethanfrey) [inline README.md:44]: Do we want to remove this, as we plan to go to 1.0? At least that reference to very old changes

File: README.md [hunk 2]
```diff
@@ -129,53 +108,52 @@ To compile all the contracts, run the following in the repo root:
 docker run --rm -v "$(pwd)":/code \
   --mount type=volume,source="$(basename "$(pwd)")_cache",target=/code/target \
   --mount type=volume,source=registry_cache,target=/usr/local/cargo/registry \
-  cosmwasm/workspace-optimizer:0.12.6
+  cosmwasm/workspace-optimizer:0.12.8
```
Comment (ethanfrey) [inline README.md:111]: Or does 0.12.9 work now?

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Contributing Guidelines
```
Comment (ethanfrey) [inline CONTRIBUTING.md:1]: Nice guidelines

**Review by ethanfrey — APPROVED** [review]
I like it.
Minor comments below

File: README.md (hunk 1 above)
Comment (uint) [reply README.md:44]: Yeah, let's get rid of that actually

**Pull request merged**